.github/workflows/test.yml

```diff
@@ -20,34 +20,25 @@ jobs:
           profile: minimal
           toolchain: ${{ matrix.rust }}
           override: true
-      - name: test/debug
+      - name: check/benches
         uses: actions-rs/cargo@v1
         with:
-          command: test
-      - name: test/release
+          command: check
+          args: --benches
+      - name: check
         uses: actions-rs/cargo@v1
         with:
-          command: test
-          args: --release
-      - name: test/debug all features
-        if: ${{ matrix.rust != 'stable' }}
+          command: check
+      - name: test/debug
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all-features
-      - name: test/release all features
-        if: ${{ matrix.rust != 'stable' }}
+      - name: test/release
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --release --all-features
+          args: --release
```

Comment:
You should keep `--all-features` in; it's more future proof

Reply:
I suppose. My thought was that the addition of later features would mean the default feature set wouldn't be separately tested, but agree that this would ensure more complete testing.